src/components/card/card.css

```diff
@@ -138,3 +155,110 @@
 	flex: 1;
 	flex-wrap: wrap;
 }
+
+/* Vertical cards */
+
+.card--vertical {
+	max-width: 33rem;
+}
+
+.card--vertical .card__content {
+	flex-flow: column wrap;
+	text-align: left;
+}
+
+/* ---ribbon ---- */
+
+.card__badge {
+	display: none;
+}
+
+.card--badge .card__badge {
+	display: block;
+}
+
+.ribbon {
+	position: absolute;
+	right: -5px;
+	top: -5px;
+	z-index: 1;
+	overflow: hidden;
+	width: 75px;
+	height: 75px;
+	text-align: right;
```

Comment:
convert to rem

Reply:
Thanks for mentioning it. I missed this.